Tests/Controller/CRUDControllerTest.php

```diff
@@ -126,7 +126,7 @@ protected function setUp()
         $this->pool = new Pool($this->container, 'title', 'logo.png');
         $this->pool->setAdminServiceIds(array('foo.admin'));
         $this->request->attributes->set('_sonata_admin', 'foo.admin');
-        $this->admin = $this->createMock('Sonata\AdminBundle\Admin\AdminInterface');
+        $this->admin = $this->createMock('Sonata\AdminBundle\Admin\AbstractAdmin');
```

Comment:
`createMock` will not be enough here… instead, you should use the mock builder and disable constructor arguments. See `getMockBuilder()` and `disableConstructorArguments()`. This should fix tests for old versions of php